smallLibraries/Google.Cloud.Templates/content/gcpmvc/CSharp/MyGcpMvcProject.csproj

```diff
@@ -0,0 +1,83 @@
+    <TargetFrameworks>netcoreapp2.1;netcoreapp2.0;netcoreapp1.1;netcoreapp1.0</TargetFrameworks>
+    <DefineConstants Condition="$(TargetFramework) == 'netcoreapp2.1'">Framework2;Framework2_1</DefineConstants>
+    <DefineConstants Condition="$(TargetFramework) == 'netcoreapp2.0'">Framework2;Framework2_0</DefineConstants>
+    <DefineConstants Condition="$(TargetFramework) == 'netcoreapp1.1'">Framework1;Framework1_1</DefineConstants>
+    <DefineConstants Condition="$(TargetFramework) == 'netcoreapp1.0'">Framework1;Framework1_0</DefineConstants>
+  </PropertyGroup>
+  <PropertyGroup Condition="'false' == 'true'">
+    <!--#endif-->
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <None Update="wwwroot\**\*;Views\**\*;Areas\**\Views">
+      <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
+    </None>
+  </ItemGroup>
+
+  <ItemGroup>
+    <!--#if (false)-->
+  </ItemGroup>
+  <ItemGroup Condition="$(TargetFramework) == 'netcoreapp1.0'">
+    <!--#endif-->
+    <!--#if (Framework1_0)-->
+    <PackageReference Include="Google.Cloud.Diagnostics.AspNetCore" Version="1.0.0" />
+    <PackageReference Include="Microsoft.AspNetCore" Version="1.0.5" />
+    <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="1.0.4" />
+    <PackageReference Include="Microsoft.AspNetCore.StaticFiles" Version="1.0.3" />
+    <PackageReference Include="Microsoft.VisualStudio.Web.BrowserLink" Version="1.0.1" />
+    <!--#endif-->
+    <!--#if (false)-->
+  </ItemGroup>
+  <ItemGroup Condition="$(TargetFramework) == 'netcoreapp1.1'">
+    <!--#endif-->
+    <!--#if (Framework1_1)-->
+    <PackageReference Include="Google.Cloud.Diagnostics.AspNetCore" Version="1.0.0" />
+    <PackageReference Include="Microsoft.AspNetCore" Version="1.1.2" />
+    <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="1.1.3" />
+    <PackageReference Include="Microsoft.AspNetCore.StaticFiles" Version="1.1.2" />
+    <PackageReference Include="Microsoft.VisualStudio.Web.BrowserLink" Version="1.1.2" />
+    <!--#endif-->
+    <!--#if (false)-->
+  </ItemGroup>
+  <ItemGroup Condition="$(TargetFramework) == 'netcoreapp2.0'">
+    <!--#endif-->
+    <!--#if (Framework2_0)-->
+    <PackageReference Include="Google.Cloud.Diagnostics.AspNetCore" Version="2.0.1" />
+    <PackageReference Include="Microsoft.AspNetCore.All" Version="2.0.0" />
+    <!--#endif-->
+    <!--#if (false)-->
+  </ItemGroup>
+  <ItemGroup Condition="$(TargetFramework) == 'netcoreapp2.1'">
+    <!--#endif-->
+    <!--#if (Framework2_1)-->
+    <PackageReference Include="Google.Cloud.Diagnostics.AspNetCore" Version="2.0.1" />
+    <PackageReference Include="Microsoft.AspNetCore.App" />
+    <PackageReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Design" Version="2.1.0" PrivateAssets="All" />
+    <!--#endif-->
+  </ItemGroup>
+
+  <Target Name="NpmInstall" BeforeTargets="Build" Inputs="package.json" Outputs="node_modules">
```

Comment:
If I do "dotnet new mvc" now, I don't get this, or PrepublishScript, or the DotNetCliToolReference for BundlerMinifier.Core. We should check it's all needed.